Create GitHub actions

nwm516 opened the pull request:
Resolves #44 

Integrated ci.yml, on-new-branch.yml, and on-pull-request.yml.

nwm516 committed:
Create GitHub workflow integrations

Integrate .yml files for SmartSheets within Shift Scheduling App repo

nwm516 committed:
Update package.json

Included "compile: tsc" script clause

nwm516 committed:
Update node-modules and package-lock.json

Updated node-modules and package-lock.json due to peer dependency issues

nwm516 committed:
Merge branch 'main' into shift-app-expo_16_44_create-github-actions

IsaacJrTypes commented:
Looks like the TS compiler is failing with github actions. I'll investigate the issue

nwm516 committed:
Merge branch 'main' into shift-app-expo_16_44_create-github-actions

nwm516 committed:
Update ExternalLink.tsx

Update ExternalLink in component folder to align with expo-router's expected types

nwm516 committed:
Update index.tsx

Commented out supabase in app-example/index.tsx

nwm516 committed:
Add unit:tests script for CI

Added unit:tests script into package.json

nwm516 committed:
Update .gitignore and package-lock.json

Added clause for coverage reports to .gitignore. Removed carats from package-lock.json.

nwm516 committed:
Create package-lock.json

Create package-lock.json

nwm516 committed:
Update package.json

Added test:coverage line to package.json
